lib/modem_info/modem_info.c

```diff
@@ -106,6 +106,8 @@ LOG_MODULE_REGISTER(modem_info);
 
 #define ICCID_PARAM_INDEX	3
 #define ICCID_PARAM_COUNT	4
+#define ICCID_LEN		    20
```

Comment:
Alignment seems messed up.

Reply:
Still doesn't look right (please open the link, it's easier to spot in a normal view than in a diff):
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/659c122cda637eab4ef2769cea89acc80fb376ee/lib/modem_info/modem_info.c#L109

Reply:
looks ok to me... unless you don't want it aligned with the other iccid values above

![image](https://user-images.githubusercontent.com/4138391/79957783-f57e0a80-8436-11ea-9a36-5307db4c4b7e.png)

Reply:
Have you configured 8 spaces per tab in your editor?

Reply:
i thought i had, but this PR may indicate things have changed :(
that screen shot is from the web view, does that not display correctly?

Reply:
I'm a bit confused now, this is how it looks in my browser (same as in my vscode):
![image](https://user-images.githubusercontent.com/22167799/79958680-84187900-8483-11ea-8a6b-ce38c62c0bcb.png)

I thought display settings for the code in the repo are global...

Reply:
![image](https://user-images.githubusercontent.com/51139291/79979885-876f2d00-84a2-11ea-8833-ad0ecb88ad0c.png)

Not sure about other editors, but if using VS code, there is a useful "Toggle render whitespace" command for this.
Looks like your editor switched over to using spaces at some point

Reply:
@bjda nice, i also noticed there is a trim trailing whitespace setting 👍 

ok my editor settings should be correct... it looks aligned there, but not on github web 🤷‍♂️